packages/dataviews/src/components/dataviews/stories/index.story.js

```diff
@@ -73,6 +89,43 @@ export const FieldsNoSortableNoHidable = ( props ) => {
 	);
 };
 
+export const CombinedFields = ( props ) => {
+	const [ view, setView ] = useState( {
+		...DEFAULT_VIEW,
+		fields: [ 'theme', 'requires', 'tested' ],
+	} );
+	const { data: shownData, paginationInfo } = useMemo( () => {
+		return filterSortAndPaginate( themeData, view, themeFields );
+	}, [ view ] );
+
+	const defaultLayouts = {
+		table: {
+			layout: {
+				combinedFields: [
```

Comment:
There's something wrong here that prevents the combined field from working properly:

https://github.com/user-attachments/assets/71f1f2c5-ebaa-4f61-a90a-5439ec86ad00

cc @youknowriad @jorgefilipecosta in case you have thoughts.